js-intermediate.js

```diff
@@ -84,40 +62,12 @@ var middleLetters1 = "hello"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
-//find the middle index and return value
-
-console.log(middleLetters1[Math.round((middleLetters1.length - 1) / 2)])
 
-//this log returns "oc" for middleLetters2
-console.log(middleLetters2[Math.floor((middleLetters2.length - 1) / 2)] + (middleLetters2[Math.round((middleLetters2.length - 1) / 2)]))
 
 
-
-//this function only returns "c" for middleLetters2
-const middleL = (string) => {
-  return string[Math.round((string.length - 1) / 2)]
-}
-
-console.log(middleL(middleLetters1))
-console.log(middleL(middleLetters2))
+// 7. Write a program to get the area of a sphere using object classes. Create three spheres with different radi as test cases. Area of a sphere =  4πr^2 (four pi r squared)
 
 
-// 7. Write a program to get the area of a sphere using object classes. Create three spheres with different radi as test cases. Area of a 
-//sphere =  4πr^2 (four pi r squared)
-
```

Comment:
The goal here is to create a class Sphere that has three instances of the class (objects)